src/js/binaryen.js-post.js

```diff
@@ -1900,10 +1917,11 @@ function wrapModule(module, self) {
   };
   self['emitBinary'] = function(sourceMapUrl) {
     return preserveStack(function() {
-      Module['_BinaryenModuleAllocateAndWrite'](temp, module, strToStack(sourceMapUrl));
-      var binaryPtr    = HEAPU32[ temp >>> 2     ];
-      var binaryBytes  = HEAPU32[(temp >>> 2) + 1];
-      var sourceMapPtr = HEAPU32[(temp >>> 2) + 2];
+      var tempBuffer = stackAlloc(12);
```

Comment:
where does this 12 come from?

Reply:
I guess it's total size of 3 pointers called `binaryPtr`, `binaryBytes` and `sourceMapPtr`

Reply:
Yeah, this relates to https://github.com/WebAssembly/binaryen/blob/master/src/binaryen-c.cpp#L2632 respectively https://github.com/WebAssembly/binaryen/blob/master/src/binaryen-c.h#L822